web/src/utils/filter-definitions.ts

```diff
@@ -251,16 +250,16 @@ export const getFilterDefinitions = (
       | undefined = undefined;
 
     if (d.id.includes('namespace')) {
-      getOptions = cap10(getNamespaceOptions);
+      getOptions = getNamespaceOptions;
```

Comment:
is `cap10` replaced by something equivalent somewhere else? I couldn't find that
How does it behave with an extremely long list of items?

Reply:
I the video recording above, you can see when I select `namespace` a scrollbar on the right when the input field is cleared